chart/values.schema.json

```diff
@@ -2397,6 +2397,30 @@
                         }
                     }
                 },
+                "podDisruptionBudget": {
+                    "description": "Webserver pod disruption budget.",
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "enabled": {
+                            "description": "Enable pod disruption budget.",
+                            "type": "boolean",
+                            "default": false
+                        },
+                        "config": {
+                            "description": "Disruption budget configuration.",
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                                "maxUnavailable": {
+                                    "description": "Max unavailable pods for webserver.",
+                                    "type": "integer",
```

Comment:
This can be an int or a string (e.g. `50%`).

Reply:
To keep consistency, I copied the properties from the PDB for the scheduler (https://github.com/apache/airflow/blob/592b46dcad1a2d13e60a49623908e89c161effea/chart/values.schema.json#L1598-L1603)

I can change both if needed

Reply:
I see. Yes, let's change both, keep the `config` and `toyaml` it in just like you are, and add `minAvailable` to the schema on both.

Reply:
Ok, added.